hocbf_controller:
  ros__parameters:
    goal_ee_pos:
    - 0.3599
    - -0.2486
    - 0.3251
    gamma_js: 10.0
    beta_js: 15.0
    d_margin: 0.0
    output_data_basename: scenario_0
    goal_maintolerance_m: 0.02
    goal_settle_time_s: 1.5
    max_sim_duration_s: 60.0
obstacles:
- name: obstacle_0
  pose_start:
    position:
    - -0.4922
    - 0.0325
    - 0.5113
  pose_end:
    position:
    - -0.4922
    - 0.0325
    - 0.5113
  size:
    radius: 0.0077
  velocity:
    linear:
    - 0.0
    - 0.0
    - 0.0
